Model/Payment.php

```diff
@@ -335,8 +351,7 @@ public function capture(InfoInterface $payment, $amount)
             $capturedData = [
                 'transaction_id' => $realTransactionId,
                 'amount'         => $captureAmount,
-                'currency'       => $order->getOrderCurrencyCode(),
-                'skip_hook_notification' => true
```

Comment:
why removing?

Reply:
@daisy1754 
The purpose of this was to avoid duplicate processing, I guess. And it's a sort of a hack. It can lead to an error if the connection breaks after the capture is done on Bolt and before the response is received in Magento. Now the record of each capture (its ID) is kept within the payment instance after it was processed. No duplicates possible.